pubspec.yaml

```diff
@@ -66,13 +66,23 @@ dependencies:
   # Allow using feather icons (nice outlined icons)
   flutter_feather_icons: ^1.0.2
 
+
```

Comment:
What's with the empty line here? There should be only one line between packages.